pkg/router/smi.go

```diff
@@ -16,7 +16,6 @@ import (
 
 	flaggerv1 "github.com/weaveworks/flagger/pkg/apis/flagger/v1beta1"
 	smiv1alpha1 "github.com/weaveworks/flagger/pkg/apis/smi/v1alpha1"
-	smiv1alpha2 "github.com/weaveworks/flagger/pkg/apis/smi/v1alpha2"
```

Comment:
Please undo the changes to this file. The conversion from alpha1 to alpha2 was developed some time ago but delayed due to Linkerd being stuck on v1alpha1, when Linkerd switches to v1alpha2 they we can activate the converter.